apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -52,6 +53,7 @@ import {
 import useTableState from "~/components/Table/ApiManagedTable/useTableState";
 import {
   stringToEnumCandidateExpiry,
+  stringToEnumCandidateSuspended,
```

Comment:
I think this column needs to be added to the table, too, right? As per the AC:

>  Admins can show a column which displays the suspendedAt status, and sort by that column

Reply:
oh you're right, while I am at it, I might as well replace Availability with this while I am at it?
one less place to have to come back to later

![image](https://user-images.githubusercontent.com/40485260/225426468-ef622f8c-3ce8-4391-9a07-10ceaecff3dc.png)

Reply:
If it's not too messy a change, sure.

Reply:
okay, done in like, the last 6 commits 😃 

sorting is odd because the selected sort is the tertiary level one, status/priority are first and locked
since this is the API driven table, I had to sort by suspended_at values, which should be fine for our purposes